project_templates/stack_package/example/ups/example.table

```diff
@@ -1,9 +1,14 @@
 # List EUPS dependencies of this package here
 # - Common third-party packages (boost, python, doxygen) and low-level
-#   LSST packages can be assumed to be recursively included by low-level
+#   LSST packages can be assumed to be recursively included by
 #   LSST packages such as utils or daf_base.
 # - Any package whose API is used should be listed explicitly
 #   rather than assuming it will be included recursively.
+# - The base package provides lsstimport. You can remove this explicit
+#   dependency if this package also declares higher-level dependencies.
+# - The utils package supports unit tests.
```

Comment:
Can say that utils can be removed if the tests are not using the memory checkers.

Reply:
...or the special asserts and temporary file helpers `utils` also provides, which really are quite useful.  We might not want to mention the memory-checkers at all, as removing them (along with Citizen) has already been RFC-approved (albeit not implemented).